spec/requests/api/user_can_finalize_order_spec.rb

```diff
@@ -0,0 +1,32 @@
+RSpec.describe 'PUT /api/orders/:id', type: :request do
+  let(:test_user) { create(:user) }
+  let(:auth) { test_user.create_new_auth_token }
+  let(:test_order) { create(:order, user: test_user) }
```

Comment:
no need to name it `test_order` or `test_user`, you are in test environment so it is stating the obvious